File: lua/refactoring/tests/refactor/106/cpp/prompt_func_param_no_return/extract.expected.cpp
#include <stdio.h>

void foo(int a) {
  int test_other = 1;

  for (int idx = 0; idx < test_other; idx++) {
    printf("%d %d", idx, a);
  }
}


void simple_function(int a) {
  int test = 1;
  foo(a);
}

File: lua/refactoring/tests/refactor/106/go/prompt_func_param_no_return/extract.expected.go
package main

import "fmt"

func foo_bar(a int, test int, test_other int) {
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
}

func simple_function(a int) {
    var test int = 1
    test_other := 1
    foo_bar(a, test, test_other)
}
